plugins/woocommerce/src/Internal/DataStores/Orders/OrdersTableDataStore.php

```diff
@@ -932,7 +932,14 @@ public function search_orders( $term ) {
 		 * @param int[]  $order_ids Search results as an array of order IDs.
 		 * @param string $term      The search term.
 		 */
-		return array_map( 'intval', (array) apply_filters( 'woocommerce_cot_shop_order_search_results', $order_ids, $term ) );
+		$order_ids = array_map( 'intval', (array) apply_filters( 'woocommerce_cot_shop_order_search_results', $order_ids, $term ) );
+
+		/**
+		 * Adding actual compatibility as well with CPT's woocommerce_shop_order_search_results.
```

Comment:
Not essential we change this ... but it almost reads a bit like a commit message rather than a description of what the hook is there for. Could a description like this be more useful to developers:

```suggestion
		 * Provides an opportunity to override the order IDs returned during an order search.
		 *
		 * @param int[]  $order_ids The order IDs to be returned.
		 * @param string $term      The search term.
```

*Or,* what about documenting the original hook and marking this one as a [duplicate](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#4-1-duplicate-hooks)?

```suggestion
		 * This filter is documented in ../../../../includes/data-stores/class-wc-order-data-store-cpt.php
```